here-naksha-app-service/src/test/java/com/here/naksha/app/service/ViewHandlerTest.java

```diff
@@ -0,0 +1,167 @@
+import com.here.naksha.app.common.ApiTest;
+import com.here.naksha.app.common.NakshaTestWebClient;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.http.HttpResponse;
+import java.util.UUID;
+
+import static com.here.naksha.app.common.CommonApiTestSetup.*;
+import static com.here.naksha.app.common.TestUtil.loadFileOrFail;
+import static com.here.naksha.app.common.assertions.ResponseAssertions.assertThat;
+
+public class ViewHandlerTest extends ApiTest {
+
+    private static final NakshaTestWebClient nakshaClient = new NakshaTestWebClient();
+    private static final String SPACE_ID = "mod-dev:topology-view";
+    private static final String DELTA_CONFIGURED_SPACE = "mod-dev:topology-delta";
+    private static final String DLB_CONFIGURED_SPACE = "mod-dev:topology-dlb";
+    private static final String BASE_CONFIGURED_SPACE = "mod-dev:topology-base";
+
+    @BeforeAll
+    static void setup() throws URISyntaxException, IOException, InterruptedException {
+        //create storages
+        createStorage(nakshaClient, "ViewHandler/setup/create_storage_sfw.json");
+        createStorage(nakshaClient, "ViewHandler/setup/create_storage_mod_dlb.json");
+        createStorage(nakshaClient, "ViewHandler/setup/create_storage_mod_delta.json");
+        createStorage(nakshaClient, "ViewHandler/setup/create_storage_mod_view_dev.json");
+
+        //create handlers
+        createHandler(nakshaClient, "ViewHandler/setup/create_handler_sfw.json");
+        createHandler(nakshaClient, "ViewHandler/setup/create_handler_mod_dlb.json");
+        createHandler(nakshaClient, "ViewHandler/setup/create_handler_mod_delta.json");
+        createHandler(nakshaClient, "ViewHandler/setup/create_handler_view_handler.json");
+
+        //create spaces
+        createSpace(nakshaClient, "ViewHandler/setup/create_space_sfw.json");
+        createSpace(nakshaClient, "ViewHandler/setup/create_space_mod_dlb.json");
+        createSpace(nakshaClient, "ViewHandler/setup/create_space_mod_delta.json");
+        createSpace(nakshaClient, "ViewHandler/setup/create_view_space.json");
+
+        //setup data
+        final String deltaBodyJson = loadFileOrFail("ViewHandler/setup/create_features_delta.json");
+        nakshaClient.post("hub/spaces/" + DELTA_CONFIGURED_SPACE + "/features", deltaBodyJson, UUID.randomUUID().toString());
+        final String dlbBodyJson = loadFileOrFail("ViewHandler/setup/create_features_dlb.json");
+        nakshaClient.post("hub/spaces/" + DLB_CONFIGURED_SPACE + "/features", dlbBodyJson, UUID.randomUUID().toString());
+        final String baseBodyJson = loadFileOrFail("ViewHandler/setup/create_features_base.json");
+        nakshaClient.post("hub/spaces/" + BASE_CONFIGURED_SPACE + "/features", baseBodyJson, UUID.randomUUID().toString());
+    }
+
+    @Test
+    void tc5001_createFeatureByViewHandler() throws Exception {
+
+        // Given:
```

Comment:
It would be good to provide one liner comment about what is happening in Given, When and Then, so someone new looking at the test can get some quicker understanding.